csi.proto

```diff
@@ -100,6 +100,51 @@ service Controller {
     returns (ControllerGetVolumeResponse) {
         option (alpha_method) = true;
     }
+
+  rpc CreateVolumeGroup(CreateVolumeGroupRequest)
```

Comment:
Add a comment that volume can't be deleted individually when it is part of the group. It has to be removed from the group first.
Delete a volume group will delete all volumes in the group.